### PR TITLE
Weather and quote

### PR DESCRIPTION
**Pull request by bhushannaikaroskar**
Hey folks please review my PR for weather API and quotes change on refresh.

Ignore /data folder. it contains quotes array data.

Live link: https://comforting-pie-232a86.netlify.app

### CODE REVIEW

File: src/components/weather/Weather.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    const [city, setCity] = useState("");
+    const [weather, setWeather] = useState(null);
+
+    const weatherApi = () => {
+        axios
+            .get(
+                `https://api.openweathermap.org/data/2.5/weather?lat=${coordinates.lat}&lon=${coordinates.lon}&appid=${process.env.REACT_APP_WEATHER_API_KEY}`
+            )
+            .then((res) => {
+                console.log(res);
```
Comment (itsmebipul00) [inline src/components/weather/Weather.jsx]: remove console logs

File: src/components/weather/Weather.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                    temperature: (res.data.main.temp - 273.15).toFixed(0),
+                    icon: res.data.weather[0].icon,
+                    city: res.data.name
+                });
+            });
+    };
+
+    const getLocation = () => {
+        navigator.geolocation.getCurrentPosition((position) => {
+            console.log(position);
```
Comment (itsmebipul00) [inline src/components/weather/Weather.jsx]: here too

**Review by itsmebipul00 — APPROVED** [review]
good to merge

File: src/components/quote/randomQuote.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    transition: all 5s ease ;
+}
+
+.quote-message{
+    transition: all 3s ease-in;
+}
+
+.quote-author{
+    /* display: none; */
+    visibility: hidden;
```
Comment (amartya-rs) [inline src/components/quote/randomQuote.css:17]: remove commented code if not required.

**Review by amartya-rs — APPROVED** [review]
Everything looks fine to me. Good to merge.

**Pull request merged**